src/CloudNative.CloudEvents/Extensions/Sampling.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright 2021 Cloud Native Foundation. 
+// Licensed under the Apache 2.0 license.
+// See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace CloudNative.CloudEvents.Extensions
+{
+    public static class Sampling
+    {
+        public static CloudEventAttribute SampledRateAttribute { get; } =
```

Comment:
Does this need to be public?

Reply:
I think it's potentially useful for consumers - it would feel odd exposing AllAttributes but not the individual ones.